ee/clickhouse/sql/dead_letter_queue.py

```diff
@@ -46,7 +46,11 @@
     ttl_period=ttl_period("_timestamp", 4),  # 4 weeks
 )
 
-KAFKA_DEAD_LETTER_QUEUE_TABLE_SQL = lambda: DEAD_LETTER_QUEUE_TABLE_BASE_SQL.format(
+# We also set kafka_max_block_size to ensure that we allow skipping all broken messages in a block
+# while ClickHouse doesn't update kafka_skip_broken_messages to be a boolean
+KAFKA_DEAD_LETTER_QUEUE_TABLE_SQL = lambda: (
+    DEAD_LETTER_QUEUE_TABLE_BASE_SQL + "SETTINGS kafka_max_block_size=65505, kafka_skip_broken_messages=65505"
```

Comment:
We'll need to add the `kafka_skip_broken_messages` everywhere, planning to do so next week. 

However why do we need to change max_block_size here? Isn't e.g. 10 or 100 sufficient? Or is the current topic just that messed up everywhere?

Reply:
Well so the max block size here is our current default. This here doesn't actually change it. I have it here so that we're very explicit. 

Essentially, `kafka_skip_broken_messages` should just be a boolean (as recognized by people in the ch repo), but it's an int of max messages to skip per block. If we want it to operate as a boolean, we need to allow skipping as many messages as there are in the block, hence the block size is set here too to be explicit.

Particularly with regards to the DLQ, I think it makes sense to skip any broken message.

Reply:
<img width="924" alt="Screenshot 2022-04-27 at 09 15 41" src="https://user-images.githubusercontent.com/38760734/165485239-e99d3a68-9f3e-4041-b677-19cfeb30f31d.png">